### DISCUSSION TITLE
🔗 Template Conditions by Product Type (Duplicate)

### DISCUSSION THREAD

**Discussion by agenciaRSE** [elementor/elementor#18311] [Feature Request]
Hi! I'm using Elementor Pro and I didn't see this feature I think it's useful, at least in my case. I need the possibility to set up a conditional based on Product Types. Depending on the Product is simple or Variable, Booking, etc. display different Templates.

Thank you.

**Comment by bluantinoo** [comment]
that's a duplicate of what? can't find the other one
